wip-trampoline.md

```diff
@@ -187,6 +187,9 @@ themselves.
 
 This grows the maximum route length to `9 * 8 = 72` total hops.
 
+The same kind of mechanism can be used to easily unlock rendezvous routing by
+simply defining a new TLV type (that will extend the `trampoline_onion` type).
```

Comment:
Uhm, actually the rendez-vous routing as proposed [here](https://github.com/lightningnetwork/lightning-rfc/wiki/Rendez-vous-mechanism-on-top-of-Sphinx) works fine without the need to extend the `trampoline_onion`. It just needs a `switch_ephemeral_key` TLV entry and we're good to go :wink:

Reply:
That `switch_ephemeral_key` is a cute trick, good job ;)
With trampoline we're able to do rendezvous without switching the ephemeral key. But your proposal with `switch_ephemeral_key` works great too and might require less space.